src/pojo-interfaces.test.ts

```diff
@@ -0,0 +1,332 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import { describe, it, expect } from "@jest/globals";
+import { dataset } from "@rdfjs/dataset";
+import * as fc from "fast-check";
+import { DataFactory } from "n3";
+import {
+  serializeBoolean,
+  serializeDatetime,
+  serializeDecimal,
+  serializeInteger,
+  xmlSchemaTypes,
+} from "./datatypes";
+import { fromRdfJsDataset, toRdfJsDataset } from "./pojo-interfaces";
+
+describe("fromRdfJsDataset", () => {
+  const fcNamedNode = fc.webUrl().map((url) => DataFactory.namedNode(url));
```

Comment:
Oh good one, I didn't realise they weren't included by default. And I think query parameters should be enabled too, since e.g. `?ext=acl` is also a Resource's name. 28f34221